meta/devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,28 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu:
+  model: MediaTek MT7628AN
+  cores: 1
+  mhz: 580
+memory_mb:
+  - 64
+storage:
+  - type: unknown
+    size_mb: 8
+device_type: WiFi Router
```

Comment:
Hmm, with OpenWrt every device with radio is (=can by) a WiFi Router. To me this looks like it will narrow down to the intended use by the vendor, and I'm not sure whether this is really a helpful/sharp information to store.

Reply:
That's true, it could be dropped

Reply:
I strongly suggest to keep the device_type. Users searching for a Wifi router do so for a reason, and they are not interested in Cameras or wired routers only.

Reply:
But what is a WiFi router?

Would a "wired router" and a "switch" be different here, when OpenWrt can have both of them be both?

I do see that Camera or NAS are categories one can work with, but it's quite hard for devices that "only do something with ethernet/Wifi" ...

But please tell me when I'm nitpicking too much here and everybody except me knows that "Wifi router" means a box with Wifi and ethernet ports that is no Camera or NAS.

Reply:
I think these can be considered as *consumer* categories. You can use any router for bitcoin mining but the initially designed purpose is maybe something else. For sorting this category seems fine to me.